cdf-core/cdf/js-modules/components/filter/models/SelectionTree.js

```diff
@@ -11,11 +11,10 @@
 
 define([
     'amd!cdf/lib/underscore',
-    '../base/BaseFilter',
     'cdf/lib/BaseSelectionTree'],
-    function( _, BaseFilter, BaseSelectionTree ) {
+    function( _, BaseSelectionTree ) {
 
-  BaseFilter.Models.SelectionTree = BaseSelectionTree.extend({
+  var SelectionTree = BaseSelectionTree.extend({
```

Comment:
You could just do return BaseSelectionTree.extend({ ... }); no need to set local variables in these cases.